packages/react-components/react-search/README.md

```diff
@@ -3,3 +3,35 @@
 **React Search components for [Fluent UI React](https://react.fluentui.dev/)**
 
 These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+SearchBox is an input designated for the purpose of search.
```

Comment:
You should expand on this a bit. It's not clear to me why I'd use this over `Input`. [`react-select` has a good example](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-select/README.md).